daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-uppercase.md

```diff
@@ -10,7 +10,7 @@ The uppercase [HTTP middleware]({{< ref middleware-concept.md >}}) converts the
 
 ## Component format
 
-In the following definition, the maximum requests per second are set to 10:
+In the following definition, it make content of request body into uppercase:
```

Comment:
```suggestion
The following definition will update all request body content into uppercase:
```

Reply:
Can't accept this suggestion as it came from another commit via the rebase from v1.0 to v1.1. I don't want a merge conflict with my other PR :)